src/smc-hub/kucalc/quota.ts

```diff
@@ -269,6 +273,35 @@ exports.quota = function (
     }
   }
 
+  // always_running – deal with it just like with member_hosting
+  if (max_upgrades.always_running == 0) {
+    quota.always_running = false;
+  } else if (!quota.always_running) {
+    if (settings.always_running) {
+      // free admin-set
+      quota.always_running = true;
+    } else {
+      // paid by some user
+      for (const userid in users) {
+        const val = users[userid];
+        if (val != null && val.upgrades && val.upgrades.always_running) {
+          quota.always_running = true;
+          break;
+        }
+      }
+      // or some site license
+      if (!quota.always_running && site_license != null) {
+        for (const license_id in site_license) {
+          const val = site_license[license_id];
+          if (val != null && val.always_running) {
```

Comment:
Cool -- I'll be able to support the new quota format for site licenses by just changing this one line.